README.md

```diff
@@ -0,0 +1,32 @@
+# Masters-way app 
+
+Multi-user application for tracking the educational process.
+
+### Modules
+* docs - project documentation
+* mw-webapp - front end for clients
```

Comment:
Who is client? Do we have frontend for developers or management?

Reply:
Change to mw-webapp - front end